2024-07-26 11:32:01.279 Python[2484:102453] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to /var/folders/6w/g897ktqs5s55q3hxdq41k35w0000gn/T/org.python.python.savedState
Traceback (most recent call last):
  File "/Users/gpena/Documents/LeetCodeAlarm/alarm.py", line 5, in <module>
    music = pyglet.media.load(f'{PATH_TO_REPO}danger.mp3')
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gpena/Documents/LeetCodeAlarm/.venv/lib/python3.12/site-packages/pyglet/media/__init__.py", line 85, in load
    return _codec_registry.decode(filename, file, streaming=streaming)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gpena/Documents/LeetCodeAlarm/.venv/lib/python3.12/site-packages/pyglet/util.py", line 166, in decode
    return decoder.decode(filename, file, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gpena/Documents/LeetCodeAlarm/.venv/lib/python3.12/site-packages/pyglet/media/codecs/coreaudio.py", line 175, in decode
    return CoreAudioSource(filename, file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gpena/Documents/LeetCodeAlarm/.venv/lib/python3.12/site-packages/pyglet/media/codecs/coreaudio.py", line 56, in __init__
    err_check(ca.ExtAudioFileOpenURL(url_ref, byref(audref)))
  File "/Users/gpena/Documents/LeetCodeAlarm/.venv/lib/python3.12/site-packages/pyglet/libs/darwin/coreaudio.py", line 239, in err_check
    raise CoreAudioException(err, err_str_db.get(err, "Unknown Error"))
pyglet.libs.darwin.coreaudio.CoreAudioException: (2003334207, 'An unspecified error has occurred.')
2024-07-26 11:34:01.183 Python[2509:104125] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to /var/folders/6w/g897ktqs5s55q3hxdq41k35w0000gn/T/org.python.python.savedState
